notebook/gateway/managers.py

```diff
@@ -222,7 +232,7 @@ def _env_whitelist_default(self):
 
     @property
     def gateway_enabled(self):
-        return bool(self.url is not None and len(self.url) > 0)
+        return bool(self.urls is not None and len(self.urls) > 0)
```

Comment:
Is ```bool``` required here ?

Reply:
Yeah it was there before just changed the property from url -> urls.